ui/packages/ui-lib/src/actionable-labeled-content/ActionableLabeledContent.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button, ButtonProps, Chip } from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import LabeledContent, { LabeledContentProps } from '../labeled-content';
+
+type Props = LabeledContentProps & {
```

Comment:
Are we going to have types separately in all files? Or there are some rules where we can keep it or not?

Reply:
My feeling is that our file number is growing a lot nowadays, a lot because we always have a .types and .messages files. This is something we can discuss, but for simple use cases (e.g. one typing) I think not having a separate file is OK.